src/java_tools/junitrunner/java/com/google/testing/coverage/BUILD

```diff
@@ -77,7 +77,7 @@ genrule(
     ],
     outs = ["JacocoCoverage_jarjar_deploy.jar"],
     cmd = "\n".join([
-        "JARJAR=\"$$(mktemp)\"",
+        "JARJAR=\"$$(mktemp 2> /dev/null || mktemp -t bazel)\"",
```

Comment:
"mktemp -t bazel" actually fails on Linux. Also, I don't see why you'd need to run mktemp first. Why not run mktemp -t bazel.XXXXXXXX right away and not fall back?